lib/reporting/command_line_options.rb

```diff
@@ -8,6 +11,8 @@ def parse!(argv, out: STDOUT)
       verbose = false
       progress = true
       period = nil
+      slice = 3.hours
+      threads = 5
```

Comment:
ditto here
```suggestion
      threads = nil
```
and the default of `5` could come after the arg parsing